clients/web/src/app.js

```diff
@@ -25,7 +25,7 @@ import 'girder/stylesheets/layout/global.styl';
 import 'girder/stylesheets/layout/layout.styl';
 
 import 'bootstrap/js/modal';
-import 'girder/utilities/jQuery'; // $.girderModal
+import 'girder/utilities/JQuery'; // $.girderModal
```

Comment:
Kind of feels wrong to have to import this for a side effect... 😢 

Is there another way, or is this how you have to do it if you need to augment the jquery prototype?